src/libs/Navigation/AppNavigator/ModalStackNavigators.js

```diff
@@ -373,8 +373,8 @@ const SettingsModalStackNavigator = createModalStackNavigator([
     },
     {
         getComponent: () => {
-            const WorkspaceBankAccountPage = require('../../../pages/workspace/WorkspaceBankAccountPage').default;
```

Comment:
can you now remove the WorkspaceBankAccountPage file?

Reply:
Updated and removed